apps/emqx_dashboard/src/emqx_dashboard_schema.erl

```diff
@@ -97,7 +99,7 @@ fields("https") ->
         bind(18084)
         | common_listener_fields() ++
             exclude_fields(
-                ["fail_if_no_peer_cert"],
+                ["fail_if_no_peer_cert", "password"],
```

Comment:
Need to hide instead of removing this configuration item.

Reply:
DONE